components/lcd_touch/esp_lcd_touch_gt911/include/esp_lcd_touch_gt911.h

```diff
@@ -41,15 +41,6 @@ esp_err_t esp_lcd_touch_new_i2c_gt911(const esp_lcd_panel_io_handle_t io, const
 #define ESP_LCD_TOUCH_IO_I2C_GT911_ADDRESS          (0x5D)
 #define ESP_LCD_TOUCH_IO_I2C_GT911_ADDRESS_BACKUP   (0x14)
 
-/**
- * @brief GT911 Configuration Type
- *
- */
-typedef struct {
-    uint32_t dev_addr;  /*!< I2C device address */
-    void *user_data;    /*!< User data passed to callback */
-} esp_lcd_touch_io_gt911_config_t;
```

Comment:
Please keep this structure, only without `user_data`. It can be good for future usage without breaking changes.